vm-setup/site.yml

```diff
@@ -90,13 +89,3 @@
 
     become: true
     become_user: root
-  - name: configure OpenShift
-    block:
-      - name: Create OpenShift config files
-        shell: "env PATH={{ tools_dir }}/openshift-origin-client-tools-v{{ openshift_version }}-{{ openshift_hash }}-linux-64bit:$PATH bash -c 'oc cluster up && oc cluster down'"
-      - name: Setup OpenShift config files
```

Comment:
@jpechane Just seeing this. Why did it get removed?

Reply:
Beacuse the pod limit is 250 in 3.11